ui-manchette/src/types.ts

```diff
@@ -51,7 +51,19 @@ export type OperationalPointType = ArrayElement<PathProperties['operational_poin
 export type StyledOperationalPointType = OperationalPointType & {
   styles?: CSSProperties;
   display?: boolean;
-  onClick?: (opId: string, opRef: HTMLDivElement | null) => void;
+  onClick?: (opId: string) => void;
+};
+
+export type WaypointMenuData = {
+  activeOperationalPointId?: string;
+  waypointMenuItems?: WaypointMenuItem[];
+  waypointMenuClassName?: string;
```

Comment:
Do we really need to allow callers to customize the class name? Wouldn't the menu already have the correct size, since it's now styled in osrd-ui? If callers really want to customize something, can't they use a regular CSS selector?

Reply:
My understanding was that we want to keep the manchette/spacetime as neutral as possible. That's why I went on this path. If the team using this package wants to use the waypoint menu it can use its custom style for it to match the rest of their app design.
The `.menu` class has been added here only for the related story.
Furthermore, if the team using the menu doesn't want to keep any of the current style, they will need to invalidate all of them even if it's a css property that they may not have used. It seems pretty painful to me.

Should we have another discussion for that ?

Reply:
Right, I'm not really getting why the manchette's styling would be in osrd-ui but the manchette menu's styling could not be. I must admit, I don't see why someone would want to change the menu's appearance, because it would then become inconsistent with the manchette.

Reply:
After discussion, it seems like the consensus is to pass the whole menu component from OSRD.